### DISCUSSION TITLE
guardians of rift should 'Quick Pass' to get in, and 'Pass' to get out

### DISCUSSION THREAD

**Discussion by WiseWoodrow** [runelite/runelite#17224] [Ideas]
Very unpleasant surprise to see that a single option (which controls a bunch of other things) turns quick-pass on for rift, because there are so many things right near the exit to click that it's incredibly easy to click out.

I've set my left-click back to Pass for now, but realistically,  if I'm IN the rift I should be safeguarded from accidentally leaving, while still being able to left-click enter right when the game opens up, right?  

Seems pretty simple

**Comment by Adam-** [comment]
The preferred way to do this is to not enable the quick-pass option and instead configure per-object what you want to quick-pass via the shift right click swap menu.
